src/components/sectionfeature.scss

```diff
@@ -1,79 +1,98 @@
 @import "../styles/settings.scss";
 @import "../styles/mixins.scss";
+@import "../styles/responsive.scss";
 
 .cardsContainer {
   display: flex;
   margin: 0 40px;
   margin-bottom: 80px;
   justify-content: center;
 
-  .middleCards {
+  .middleCardsText {
     display: flex;
-    justify-content: space-between;
-    gap: 120px;
+    flex-direction: column;
+    align-items: flex-start;
+    min-width: 30%;
+    justify-content: center;
+    order: 2;
 
-    &.reversed {
-      flex-direction: row-reverse;
-    }
-
-    .middleCardsText {
-      display: flex;
-      flex-direction: column;
-      align-items: flex-start;
-      min-width: 30%;
-      justify-content: center;
-      order: 2;
-
-      .middleCardsTextSection {
-        font-family: $secondaryFontFamily;
-        font-style: normal;
-        font-weight: 800;
-        font-size: 16px;
-        line-height: 100%;
-
-        &.primary {
-          color: $firstCardTitleColor;
-        }
+    .middleCardsTextSection {
+      font-family: $secondaryFontFamily;
+      font-style: normal;
+      font-weight: 800;
+      font-size: 16px;
+      line-height: 100%;
 
-        &.secondary {
-          color: $secondCardTitleColor;
-        }
-
-        &.tertiary {
-          color: $thirdCardTitleColor;
-        }
+      &.primary {
+        color: $firstCardTitleColor;
       }
 
-      .middleCardsTextTitle {
-        @include fontStylemanropeBold;
-        text-align: initial;
-        margin-top: 12px;
-        margin-bottom: 24px;
+      &.secondary {
+        color: $secondCardTitleColor;
       }
 
-      .middleCardsTextDescription {
-        text-align: initial;
-        @include fontStyleInter;
+      &.tertiary {
+        color: $thirdCardTitleColor;
       }
     }
 
-    .middleCardsCard {
-      display: flex;
-      justify-content: flex-start;
-      width: 100%;
+    .middleCardsTextDescription {
+      text-align: initial;
+      @include fontStyleInter;
+    }
+  }
 
-      .middleCardsCardImagesContainer {
-        .middleCardsCardImagesBackground {
-          width: 100%;
-          height: 100%;
+  .middleCardsCard {
+    display: flex;
+    justify-content: flex-start;
+    width: 100%;
 
-          img {
-            height: 100%;
-            width: 100%;
-            border-radius: 12px;
-          }
+    .middleCardsCardImagesContainer {
+      .middleCardsCardImagesBackground {
+        width: 100%;
+        height: 100%;
+
+        img {
+          height: 100%;
+          width: 100%;
+          border-radius: 12px;
         }
       }
     }
   }
 }
+
+.middleCards {
+  display: flex;
+  justify-content: space-between;
+  gap: 30px;
+
+  &.reversed {
+    flex-direction: row-reverse;
+  }
+}
+
+.middleCardsTextTitle {
+  @include fontStylemanropeBold;
+  text-align: initial;
+  margin-top: 12px;
+  margin-bottom: 24px;
+}
+
+@media #{$medium-screen} {
+
+  .cardsContainer {
+    margin:40 24px;
```

Comment:
bug: this is invalid CSS, have you noticed it?

Reply:
@sandrina-p What do you mean by its invalid css? I don't get any error message and If I inspect the code, it works

Reply:
It doesn't work Clara, check the devs tools, it's invalid, it's still applying 40px instead of 24px in the x-axis.

The right way it's `40px 24px`. margins do not accept unitless values.

<img width="616" alt="image" src="https://user-images.githubusercontent.com/14869087/183112570-3bcaf83c-21bc-4603-bfd2-57291d94caa3.png">